tests/baselines/reference/user/debug.log

```diff
@@ -1,48 +1,85 @@
-node_modules/debug/src/debug.js(47,57): error TS2345: Argument of type 'string' is not assignable to parameter of type 'number'.
-node_modules/debug/src/debug.js(51,18): error TS2339: Property 'colors' does not exist on type 'typeof createDebug'.
-node_modules/debug/src/debug.js(51,50): error TS2339: Property 'colors' does not exist on type 'typeof createDebug'.
-node_modules/debug/src/debug.js(75,10): error TS2339: Property 'diff' does not exist on type '{ (...args: any[]): void; namespace: string; enabled: boolean; useColors: any; color: number; destroy: () => boolean; }'.
-node_modules/debug/src/debug.js(76,10): error TS2339: Property 'prev' does not exist on type '{ (...args: any[]): void; namespace: string; enabled: boolean; useColors: any; color: number; destroy: () => boolean; }'.
-node_modules/debug/src/debug.js(77,10): error TS2339: Property 'curr' does not exist on type '{ (...args: any[]): void; namespace: string; enabled: boolean; useColors: any; color: number; destroy: () => boolean; }'.
-node_modules/debug/src/debug.js(112,13): error TS2551: Property 'formatArgs' does not exist on type 'typeof createDebug'. Did you mean 'formatters'?
-node_modules/debug/src/debug.js(114,23): error TS2339: Property 'log' does not exist on type '{ (...args: any[]): void; namespace: string; enabled: boolean; useColors: any; color: number; destroy: () => boolean; }'.
-node_modules/debug/src/debug.js(114,38): error TS2339: Property 'log' does not exist on type 'typeof createDebug'.
-node_modules/debug/src/debug.js(120,29): error TS2339: Property 'useColors' does not exist on type 'typeof createDebug'.
-node_modules/debug/src/debug.js(125,37): error TS2339: Property 'init' does not exist on type 'typeof createDebug'.
-node_modules/debug/src/debug.js(126,13): error TS2339: Property 'init' does not exist on type 'typeof createDebug'.
-node_modules/debug/src/debug.js(153,11): error TS2339: Property 'save' does not exist on type 'typeof createDebug'.
-node_modules/debug/src/debug.js(155,3): error TS2323: Cannot redeclare exported variable 'names'.
-node_modules/debug/src/debug.js(156,3): error TS2323: Cannot redeclare exported variable 'skips'.
-node_modules/debug/src/debug.js(217,12): error TS2304: Cannot find name 'Mixed'.
-node_modules/debug/src/debug.js(218,13): error TS2304: Cannot find name 'Mixed'.
-node_modules/debug/src/index.js(6,47): error TS2339: Property 'type' does not exist on type 'Process'.
-node_modules/debug/src/node.js(26,1): error TS2323: Cannot redeclare exported variable 'colors'.
-node_modules/debug/src/node.js(31,5): error TS2323: Cannot redeclare exported variable 'colors'.
-node_modules/debug/src/node.js(60,39): error TS2345: Argument of type 'string | undefined' is not assignable to parameter of type 'string'.
+node_modules/debug/dist/debug.js(3,100): error TS2539: Cannot assign to '_typeof' because it is not a variable.
```

Comment:
These changes are *probably* the result of a major new version but I need to confirm.

Reply:
Yes, the code seems entirely different.

I'll accept these baselines for now, but I should probably sweep through at some point and make sure the new errors are mostly good. There's about 40 more than before.